src/main/java/ResourceDiscovery/AssetObjects/AssetObject.java

```diff
@@ -1,25 +1,45 @@
 package ResourceDiscovery.AssetObjects;
 
+import ResourceDiscovery.Main;
```

Comment:
why are we importing `Main`? it should be the run module, not a dependency for other modules. Instead of using the consts from Main, put them in a separate module and import from there in both Main and here.